application/package/rpm/README.md

```diff
@@ -0,0 +1,12 @@
+# What is this?
+A simple RPM package for MultiMC that wraps the contains a script that downloads and installs real MultiMC on Red Hat based systems.
```

Comment:
'wraps *the* contains' - extra word

Reply:
This one got copied from the Ubuntu README... what should I change it to?
Remove `wraps the`?

Reply:
I guess. Just make it make sense. If you can fix the other README too, that would be cool